catalog/generators/database-crud-thorntail/files/src/main/fabric8/deployment.yml

```diff
@@ -0,0 +1,23 @@
+apiVersion: v1
+kind: Deployment
+metadata:
+  name: ${project.artifactId}
+spec:
+  template:
+    spec:
+      containers:
+        - env:
+            - name: DB_USERNAME
+              valueFrom:
+                 secretKeyRef:
+                   name: my-database-secret
+                   key: user
+            - name: DB_PASSWORD
+              valueFrom:
+                 secretKeyRef:
+                   name: my-database-secret
+                   key: password
+            - name: GC_MAX_METASPACE_SIZE
+              value: "150"
+            - name: JAVA_OPTIONS
+              value: "-Dswarm.datasources.data-sources.MyDS.connection-url=jdbc:postgresql://$(MY_DATABASE_SERVICE_HOST):$(MY_DATABASE_SERVICE_PORT)/my_data -Dswarm.datasources.data-sources.MyDS.user-name=$(DB_USERNAME) -Dswarm.datasources.data-sources.MyDS.password=$(DB_PASSWORD) -Dswarm.datasources.data-sources.MyDS.driver-name=postgresql"
```

Comment:
The `MY_DATABASE_SERVICE_HOST` still needs to be changed, right?

Reply:
This file doesn't exist anymore